km/km_filesys.c

```diff
@@ -1951,27 +1962,55 @@ static inline void do_full_write(int fd, char* bufp, size_t bufl)
 static inline size_t
 fs_core_save_pipe_contents(char* buf, size_t length, km_file_t* file, int writefd, size_t queuedbytes)
```

Comment:
replace file and writefd paramenters with fd. This will make code more understandable.

Reply:
Will fix.